dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1066,333 +968,387 @@ data:
+          },
+          "pluginVersion": "10.4.1",
+          "targets": [
             {
               "datasource": {
                 "type": "prometheus",
                 "uid": "${datasource}"
               },
-              "description": "The objective for the given time window.",
-              "gridPos": {
-                "h": 7,
-                "w": 10,
-                "x": 0,
-                "y": 28
-              },
-              "id": 10,
-              "options": {
-                "code": {
-                  "language": "plaintext",
-                  "showLineNumbers": false,
-                  "showMiniMap": false
-                },
-                "content": "<h1><center>SLO Definition</center></h1>",
-                "mode": "html"
+              "refId": "A"
+            }
+          ],
+          "title": "Pipeline Execution SLO",
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "description": "The current measurement for the given time window.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "thresholds"
               },
-              "pluginVersion": "10.4.1",
-              "targets": [
+              "mappings": [
                 {
-                  "datasource": {
-                    "type": "prometheus",
-                    "uid": "${datasource}"
+                  "options": {
+                    "match": "null",
+                    "result": {
+                      "text": "N/A"
+                    }
                   },
-                  "refId": "A"
+                  "type": "special"
                 }
               ],
-              "title": "SLO",
-              "type": "text"
+              "max": 1,
+              "min": 0,
+              "noValue": "-",
+              "thresholds": {
+                "mode": "percentage",
+                "steps": [
+                  {
+                    "color": "red",
+                    "value": null
+                  },
+                  {
+                    "color": "green",
+                    "value": 95
+                  }
+                ]
+              },
+              "unit": "percentunit"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 7,
+            "w": 5,
+            "x": 10,
+            "y": 56
+          },
+          "id": 70,
+          "maxDataPoints": 100,
+          "options": {
+            "minVizHeight": 75,
+            "minVizWidth": 75,
+            "orientation": "horizontal",
+            "reduceOptions": {
+              "calcs": [],
+              "fields": "",
+              "values": false
             },
+            "showThresholdLabels": false,
+            "showThresholdMarkers": false,
+            "sizing": "auto"
+          },
+          "pluginVersion": "10.4.1",
```

Comment:
maybe the newer plugin version has bearing on some of my other comments ?

Reply:
I'm not sure but this version looks unchanged here.

Reply:
ack